start.py

```diff
@@ -280,7 +280,7 @@ def get_constants(metadata):
     )
     try:
         constants_from_json = json.loads(constants_json)
-    except Exception as e:
+    except Exception:
```

Comment:
As above, linting related changes in an unrelated commit/PR. Please move to a separate commit/PR.

Reply:
no longer in mr.